TokenSale.sol

```diff
@@ -17,28 +17,37 @@ along with the DAO.  If not, see <http://www.gnu.org/licenses/>.
 
 
 /*
-Token Sale contract, used by the DAO to raise its funding goal
+Token Sale contract, used by the DAO to sell its tokens and initialize its fund
 */
 
 import "./Token.sol";
 
 
 contract TokenSaleInterface {
-    uint public closingTime;                   // end of token sale, in Unix time
-    uint public minValue;                      // minimum funding goal of the token sale, denominated in tokens
-    bool public isFunded;                      // true if DAO has reached its minimum funding goal, false otherwise
-    address public privateSale;                // used for DAO splits - if privateSale is 0, then it is a public sale, otherwise, only the address stored in privateSale is allowed to purchase tokens
-
-    /// @dev Constructor setting the minimum funding goal and the end of the Token Sale
+    
+    // End of token sale, in Unix time
+    uint public closingTime;   
+    // Minimum funding goal of the token sale, denominated in tokens
+    uint public minValue;  
+    // True if the DAO reached its minimum funding goal, false otherwise
+    bool public isFunded;   
+    // For DAO splits - if privateSale is 0, then it is a public sale, otherwise
+    // only the address stored in privateSale is allowed to purchase tokens
+    address public privateSale;               
+    
+    /// @dev Constructor setting the minimum funding goal and the 
+    /// end of the Token Sale
     /// @param _minValue Token Sale minimum funding goal
     /// @param _closingTime Date (in Unix time) of the end of the Token Sale
+    // This function will not likely be used so it is commented out
```

Comment:
This sentence is false. This is the constructor, and since I can not overload it in the implementation, I need to comment it out. But it is used!